### ISSUE TITLE
Wheel for linux is missing in Pypi

### ISSUE THREAD

**Issue by Programmierschmied** [tennom/saxonpy#1]
Hello folk!

sadly the wheel package is missing in PyPi. Result is I can't install it in Linux easy. :-)

Best regards,
Veganizer

**Comment by tennom** [comment]
Hi Veganizer,

Thanks for filing issue! Which Linux distribution are you using now? I use Saxonpy on Linux, and if you’re using a linux that’s not built for ARM or 32bit or your Python version is 3.6 to 3.10, it should work I think. Can you share your linux architecture info and Python version, maybe I can help?
________________________________
From: Veganizer ***@***.***>
Sent: Friday, April 29, 2022 12:44:35 PM
To: tennom/saxonpy ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [tennom/saxonpy] Wheel for linux is missing in Pypi (Issue #1)


Hello folk!

sadly the wheel package is missing in PyPi. Result is I can't install it in Linux easy. :-)

Best regards,
Veganizer

**Comment by Programmierschmied** [comment]
Hello,

I am using AlpineLinux with amd64 architecture with Python version 3.10.3-r1.

Thank you very much!

**Comment by tennom** [comment]
You’re welcome. It’s probably because Alpinelinux is base on musl libc while I used gnu libc for compiling the Saxon C code. I will need to look for more info on this and give you some feedback.
________________________________

**Comment by pokoli** [comment]
I'm on archlinux with python 3.10.4 and  installing from Pypi I get 0.0.2 also: 

``` 
$ pip install saxonpy
Collecting saxonpy
  Downloading saxonpy-0.0.2-cp310-cp310-manylinux_2_12_x86_64.manylinux2010_x86_64.whl (48.0 MB)
     |████████████████████████████████| 48.0 MB 2.2 MB/s 
Installing collected packages: saxonpy
Successfully installed saxonpy-0.0.2
```

**Comment by tennom** [comment]
Yeah, I need to find some time to make a package for Alpine Linux.
________________________________

**Comment by luap42** [comment]
Hi, has there been any update with regards to providing a v0.0.3 wheel for Linux? The SaxonC version used in v0.0.2 seems to be quite unstable in some regards, which are supposed to be fixed in the one used for v0.0.3. Therefore it would be great to have the changes for Linux. :see_no_evil:

Thanks

**Comment by ond1** [comment]
Have you tried using the official SaxonC 12 on pypi? There are three products available:
saxonche (opensource): https://pypi.org/project/saxonche/
Also available the commercial products saxoncpe and saxoncee

**Comment by luap42** [comment]
@ond1 No, I hadn't found that yet, because all online resources always refer to this project AFAIK.

Thank you very much for the pointer, I'll try it out later.

**Comment by tennom** [comment]
@luap42 Good to hear from you. Yes, definitely check out the packages from @ond1. By the way, the underlying versions of the V0.02 and the V0.03 are the same. I had to use the different sub names because I automated the processes of making and uploading the Python wheel packages. I will fix all of them for the next release.
________________________________
